### PR TITLE
Added id's to footer links and Ready page web elements

### PR DESCRIPTION
**Pull request by shan14T**
### What github.com/wevote/WebApp/issues does this fix?
https://wevoteusa.atlassian.net/browse/WV-856

### Changes included this pull request?
added id's to web elements on ready page and footer section.

### CODE REVIEW

File: src/js/components/Intro/FriendInvitationOnboardingIntro.jsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ class FriendInvitationOnboardingIntro extends Component {
             </>
           )}
         </FriendInvitationTopHeader>
-        <FriendInvitationIntroHeader className="FriendInvitationIntroHeader">
+        <FriendInvitationIntroHeader id="WeVotehelpsyouListHeaderText" className="FriendInvitationIntroHeader">
```
Comment (DaleMcGrew) [inline src/js/components/Intro/FriendInvitationOnboardingIntro.jsx:156]: Hi @shan14T thank you for adding this. Please use "camel case name, lower case start" format for ids. So this should be "weVoteHelpsYouListHeaderText"

File: src/js/components/Intro/FriendInvitationOnboardingIntro.jsx [hunk 2]
```diff
@@ -184,7 +184,7 @@ class FriendInvitationOnboardingIntro extends Component {
                   Make a plan for casting your vote. See your entire ballot. Find your polling location.
                 </StepText>
               ) : (
-                <StepText onClick={this.onClickShowAllStepOne}>
+                <StepText id="BeconfidentinyourchoicesHeader" onClick={this.onClickShowAllStepOne}>
```
Comment (DaleMcGrew) [inline src/js/components/Intro/FriendInvitationOnboardingIntro.jsx:187]: For camelCase, this should id be "beConfidentInYourChoicesHeader"

File: src/js/components/Intro/FriendInvitationOnboardingValues.jsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ class FriendInvitationOnboardingValues extends Component {
               Opinions will be highlighted on your ballot based on what you follow. Follow as many values/issues...
               {' '}
               (
-              <span className="u-cursor--pointer u-link-color">
+              <span id="showmorelink" className="u-cursor--pointer u-link-color">
```
Comment (DaleMcGrew) [inline src/js/components/Intro/FriendInvitationOnboardingValues.jsx:65]: Please change id to "showMoreLink"

**Review by DaleMcGrew — APPROVED** [review]
Thank you @shan14T, these are some great updates. I have added comments about the id naming format needing to be changed to camelCase. I have specified the names in a few examples, but please update all new ids you added to the camelCase format. Thank you! I'm going to approve this pull request, so please submit a follow up pull request once you've made the improvements.

**Comment by DaleMcGrew** [comment]
👍

File: src/js/components/Intro/FriendInvitationOnboardingIntro.jsx (hunk 1 above)
Comment (shan14T) [reply src/js/components/Intro/FriendInvitationOnboardingIntro.jsx:156]: seeing this msg now. will be modifying the id's asap as per the camel casing format. thank you.

**Pull request merged**